### ISSUE TITLE
How to get started?

### ISSUE THREAD

**Issue by mdings** [pojala/electrino#16]
Say I wanted to build a concept using electrino. Just to see what I would bump into or find out some of the limitations. How do I get started?

**Comment by indrakaw** [comment]
@mdings **R**ead **T**he **F**ine **M**anual. Might it will helps you.

**Comment by mdings** [comment]
@indrakaw  And where is this **F**ine **M**anual you are talking about? The readme surely isn't enough to get one started..

**Comment by PragmaticEd** [comment]
@indrakaw Yes, where is this manual? No info on how to use this project.. The only demo project provided, also doesn't work, having error `cannot find module electrino`. Such a nice project description, while project it self seems dead and unusable...

**Comment by indrakaw** [comment]
then it's `WIP`. Please @pojala.

I use npm references for testing stuff with electrino.

**Comment by crowecawcaw** [comment]
I got this working by:

1.  Cloning the repo
2. Opening Electrino/MacOS/Electrino.xcodeproj in XCode
3. Moving the test-app and test-tray folders from /Electrino to /Electriono/MacOS
4. Running the project in XCode.
